api/helpers/db.js

```diff
@@ -440,13 +441,12 @@ function yesno(value) {
 }
 
 function date(value) {
+  if (!value) return null;
+
   if (isDate(value)) {
     return text(value);
   } else {
-    if (!value) {
-      return null;
-    }
-    return new Date(value);
+    return new Date(moment(value).format());
```

Comment:
needed to use moment here because just using new Date wasn't giving us the results we would expect.
<img width="432" alt="Screenshot 2019-06-26 09 31 37" src="https://user-images.githubusercontent.com/130878/60210633-27701180-9812-11e9-8713-af59ea4627a7.png">

Reply:
Makes sense. JavaScript date parsing with the built-in date is not well defined and inconsistently implemented. If this is the only use we have for Moment, then we could avoid the extra dependency by parsing ISO 8601 ourselves, since it is designed to be easy to manipulate.